deploy/examples/create-external-cluster-resources.py

```diff
@@ -1451,6 +1473,21 @@ def _gen_output_map(self):
                 self.out_map["CSI_CEPHFS_PROVISIONER_SECRET"],
                 self.out_map["CSI_CEPHFS_PROVISIONER_SECRET_NAME"],
             ) = self.create_cephCSIKeyring_user("client.csi-cephfs-provisioner")
+            # pin the default csi subvolumegroup
+            self.pin_subvolume(
+                "csi", self._arg_parser.cephfs_filesystem_name, "distributed", "1"
+            )
```

Comment:
You should create the subvolumegroup with the name `csi` and pin the subvolume group

Reply:
`pin_subvolume` function is calling the `self.get_or_create_subvolume_group(subvolume_group, cephfs_filesystem_name)` internally which will create the subvolume if it is not present.

Reply:
It doesn't look good to handle `get_or_create_subvolume_group` in multiple places. don't mix the creation of subvolumegroup and pinning.